first-analysis-steps/lhcb-dev.md

```diff
@@ -37,6 +41,7 @@ cumbersome, for any serious development the usage of vanilla git is much more
 stable. Please consider using it if you can spare the compilation time.
```

Comment:
It seems the link to the bottom of the page does not work.

Reply:
Also, above (I cannot leave a comment there as you haven't edited that line): the link to gitlab is not clickable.